domains/polo.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "polo-1245-oficial",
+           "email": "81937184+polo-1245-oficial@users.noreply.github.com",
+           "discord": "725316907187568701"
+        },
+    
+        "record": {
+            "TXT": "_value=vc-domain-verify=polo.is-a.dev,218d849c972de88cbdf4"
```

Comment:
TXT record alone does nothing... and we dont support vercel atm. (might soon tho)